ShopAppTests/App/Account/Controllers/AddressFormViewControllerSpec.swift

```diff
@@ -0,0 +1,277 @@
+//
+//  AddressFormViewControllerSpec.swift
+//  ShopAppTests
+//
+//  Created by Evgeniy Antonov on 2/23/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import Nimble
+import Quick
+import RxSwift
+import ShopApp_Gateway
+
+@testable import ShopApp
+
+class AddressFormViewControllerSpec: QuickSpec {
+    override func spec() {
+        var viewController: AddressFormViewController!
+        var viewModelMock: AddressFormViewModelMock!
+        var countryPicker: BasePicker!
+        var nameTextFieldView: InputTextFieldView!
+        var lastNameTextFieldView: InputTextFieldView!
+        var addressTextFieldView: InputTextFieldView!
+        var addressOptionalTextFieldView: InputTextFieldView!
+        var cityTextFieldView: InputTextFieldView!
+        var statePicker: BasePicker!
+        var zipCodeTextFieldView: InputTextFieldView!
+        var phoneTextFieldView: InputTextFieldView!
+        var submitButton: BlackButton!
+        
+        describe("when view loaded") {
+            beforeEach {
+                viewController = UIStoryboard(name: StoryboardNames.account, bundle: nil).instantiateViewController(withIdentifier: ControllerIdentifiers.addressForm) as! AddressFormViewController
```

Comment:
Why are you duplicate initialization of controller and outlets few times? We can do it one time in `beforeEach` closure before first `describe` closure with test cases.